integration-tests/cucumber-tests-platform-publiclighting/src/test/java/org/opensmartgridplatform/cucumber/platform/publiclighting/mocks/oslpdevice/MockOslpServer.java

```diff
@@ -149,15 +146,18 @@ public MockOslpServer(final CoreDeviceConfiguration configuration, final int osl
         this.responseDelayRandomRange = reponseDelayRandomRange;
     }
 
-    public Integer getSequenceNumber() {
-        return this.channelHandler.getSequenceNumber();
+    public Integer getSequenceNumber(final String deviceUID) {
```

Comment:
Proper camel-casing would be `deviceUid`. (Also elsewhere in other added methods.)

Reply:
I will refactor this